packages/suite-desktop-api/src/validation.ts

```diff
@@ -20,10 +19,10 @@ export const isObject = (shape: { [key: string]: OptionalPrimitive }, value: any
     return !keys.includes(false);
 };
 
-const validThemes: UnionToTuple<SuiteThemeVariant> = ['light', 'dark', 'system'];
+const validThemes: Array<SuiteThemeVariant> = ['light', 'dark', 'system'];
 export const isTheme = (theme: any) => validThemes.includes(theme);
 
-const validChannels: UnionToTuple<keyof RendererChannels> = [
+const validChannels: Array<keyof RendererChannels> = [
```

Comment:
@szymonlesisz Can you verify this OK? This started to throw error because not correct order of items in array. (Tuple has specific type for every index in array). From usage it seems that you don't need tuple at all and plain old array is enough.